content/master/guides/metrics.md

```diff
@@ -23,39 +23,84 @@ prometheus.io/port: "8080"
 prometheus.io/scrape: "true"
 ```    
 
+## Crossplane core metrics
+
+These metrics are emitted by the Crossplane pod itself.
+
+{{< table "table table-hover table-striped table-sm">}}
+| Metric Name | Description | Further Explanation |
+| --- | --- | --- |
+| {{<hover label="function_run_function_request_total" line="1">}}function_run_function_request_total{{</hover>}} | Total number of RunFunctionRequests sent |  |
+| {{<hover label="function_run_function_response_total" line="2">}}function_run_function_response_total{{</hover>}} | Total number of RunFunctionResponses received |  |
+| {{<hover label="function_run_function_seconds" line="3">}}function_run_function_seconds{{</hover>}} | Histogram of RunFunctionResponse latency (seconds) |  |
+| {{<hover label="function_run_function_response_cache_hits_total" line="4">}}function_run_function_response_cache_hits_total{{</hover>}} | Total number of RunFunctionResponse cache hits |  |
+| {{<hover label="function_run_function_response_cache_misses_total" line="5">}}function_run_function_response_cache_misses_total{{</hover>}} | Total number of RunFunctionResponse cache misses |  |
+| {{<hover label="circuit_breaker_opens_total" line="6">}}circuit_breaker_opens_total{{</hover>}} | Number of times the XR circuit breaker transitioned from closed to open |  |
+| {{<hover label="circuit_breaker_closes_total" line="7">}}circuit_breaker_closes_total{{</hover>}} | Number of times the XR circuit breaker transitioned from open to closed |  |
+| {{<hover label="circuit_breaker_events_total" line="8">}}circuit_breaker_events_total{{</hover>}} | Number of XR watch events handled by the circuit breaker, labelled by outcome |  |
+| {{<hover label="engine_controllers_started_total" line="9">}}engine_controllers_started_total{{</hover>}} | Total number of controllers started |  |
+| {{<hover label="engine_controllers_stopped_total" line="10">}}engine_controllers_stopped_total{{</hover>}} | Total number of controllers stopped |  |
+| {{<hover label="engine_watches_started_total" line="11">}}engine_watches_started_total{{</hover>}} | Total number of watches started |  |
+| {{<hover label="engine_watches_stopped_total" line="12">}}engine_watches_stopped_total{{</hover>}} | Total number of watches stopped |  |
+{{</table >}}
+
+## Provider metrics
+
+These metrics are emitted by Crossplane providers. All providers built with crossplane-runtime emit the `crossplane_managed_resource_*` metrics.
+
+Providers expose metrics on the `metrics` port (default `8080`). To scrape these metrics, configure a `PodMonitor` or add Prometheus annotations to the provider's `DeploymentRuntimeConfig`.
+
+{{< table "table table-hover table-striped table-sm">}}
+| Metric Name | Description | Further Explanation |
```

Comment:
same here, the "further explanation" column is empty and can be removed...

Reply:
same